.ci/bump-version.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+AGENT_VERSION="${1?Missing the APM ruby agent version}"
+
+## Use docker to bump the version to ensure the environment is easy to reproduce.
+docker run --rm -t \
+  --user $UID \
+  -e LOCAL_USER_ID=$UID \
+  -e HOME=/tmp \
+  -w /app \
+  -v "$(pwd):/app" \
+  ruby:2.6 /bin/bash -c "set -x
```

Comment:
You can get the current Ruby version from the Gemfile with 
```sh
$ cat Gemfile | grep '^ruby' | sed -E "s/.*[\"'](.+)[\"']/\1/"
2.6.5
```

Reply:
If there is a Docker image for that specific tag, which imho should be there, then it's perfect to not hardcode the Ruby version

Reply:
This docker is only in charge to bump the version, nothing else, so I'll gather the ruby version using the above command, although I might add some complexity on the other hand.

Reply:
I'm afraid the above command won't work with the opbeans-ruby as its Gemfile doesn't contain any references to ruby.

Reply:
Bundler will complain on `bundle install` if it isn't using the same version as defined by `Gemfile` so it might be necessary to be precise.

Reply:
For the time being, I'll proceed with the merge and we can revisit it if required